agent/agent.go

```diff
@@ -228,21 +228,9 @@ func New(c *Config) (*Agent, error) {
 		httpAddrs:       httpAddrs,
 		tokens:          new(token.Store),
 	}
-	if err := a.resolveTmplAddrs(); err != nil {
```

Comment:
huh yeah that looks like it never worked. Is there a use case for people wanting to pass their own tagged addresses ever, if so seems like we should open a ticket around that